releasenotes/notes/add-elide-permutations-to-pipeline-077dad03bd55ab9c.yaml

```
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    The transpiler pass :class:`~.ElidePermutations`
+    runs by default with optimization level 3. Intuitively, removing
```

2 and 3 share the optimizations run during `init` now (since #12149 )

```suggestion
    runs by default with optimization level 2 and 3. Intuitively, removing
```